moped-editor/src/views/dashboard/DashboardView.js

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from "react";
 import { useQuery } from "@apollo/client";
+import moment from "moment";
```

Comment:
I recall that moment is being deprecated. There are other libraries, and I believe it's become straight forward to do in vanilla JS. 

This is a good read on it: https://terodox.tech/migrating-away-from-momentjs-part1/ and [moment's own status](https://momentjs.com/docs/#/-project-status/) for sure.

I don't think this is enough to withhold approval at this time, but I do think a tech-debt spin-off issue may be warranted.

Reply:
Yeah, I agree with Frank on this one. For the sake of the next release, I'd suggest merging as is, but I believe we already had a spike to remove and replace `moment.js` as a dependency so it would be a step backwards to include it in the codebase again. I'll make a new issue and assign it to @tillyw as tech debt.

Reply:
https://github.com/cityofaustin/atd-data-tech/issues/9393